### PR TITLE
update clientWrapper

### PR DESCRIPTION
**Pull request by odinuv**
https://keboola.atlassian.net/browse/SOX-209

Also fixes
https://keboola.atlassian.net/browse/SOX-370

### CODE REVIEW

**Review by MiroCillik — APPROVED** [review]
Prijde mi to OK

File: composer.json [hunk 1]
```diff
@@ -23,13 +23,13 @@
         "aws/aws-sdk-php": "3.258",
         "keboola/api-error-control": "^3.10",
         "keboola/configuration-variables-resolver": "^4.0",
-        "keboola/dockerbundle": "dev-odin-library-3",
+        "keboola/dockerbundle": "dev-odin-SOX-332",
```
[the remark this answers is not in the document]
Comment (odinuv) [reply composer.json]: fixed

File: tests/Functional/DebugModeTest.php [hunk 2]
```diff
@@ -342,7 +333,7 @@ public function testDebugModeConfiguration(): void
             }
         }
         $config = json_decode(strrev((string) base64_decode($output)), true);
```
[the remark this answers is not in the document]
Comment (odinuv) [reply tests/Functional/DebugModeTest.php:335]: Protože kdyby nebylo, tak by output filter ty zašifrovaný hodnoty zahvězdičkoval, tak se to otáčí  https://github.com/keboola/job-runner/pull/232/files#diff-cd37c3711fe102e057f68c24473d7382bda387b119bc8a76ba59f3ab5d1e22edL295

[the remark this answers is not in the document]
Comment (MiroCillik) [reply tests/Functional/DebugModeTest.php:335]: ahaaa

**Pull request merged**